aerospike/datadog_checks/aerospike/check.py

```diff
@@ -1,22 +1,78 @@
 # (C) Datadog, Inc. 2022-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-from datadog_checks.base import OpenMetricsBaseCheckV2
+import re
 
-from .metrics import METRIC_MAP
+from datadog_checks.base import OpenMetricsBaseCheckV2
 
+from .metrics import METRIC_MAP, METRIC_MAP_V7
 
-class AerospikeCheckV2(OpenMetricsBaseCheckV2):
+RENAMED_LABELS = {'cluster_name': 'aerospike_cluster', 'service': 'aerospike_service'}
 
-    __NAMESPACE__ = 'aerospike'
 
+class AerospikeCheckV2(OpenMetricsBaseCheckV2):
+    __NAMESPACE__ = "aerospike"
     DEFAULT_METRIC_LIMIT = 0
 
     def __init__(self, name, init_config, instances):
-        super().__init__(name, init_config, instances)
+        super(AerospikeCheckV2, self).__init__(name, init_config, instances)
+        self.exporter_url = instances[0]["openmetrics_endpoint"]
+        self.build_version = None
 
     def get_default_config(self):
+        """
+        shares details of metrics map depending on aerospike version 7 or below
+        """
+        metric_map_to_use = self._get_metrics_map()
```

Comment:
I'm wondering if we need this logic to determine the metric map at runtime.

Is it the case that some metric in v7 has the same name as a metric in in <v7 but a different value or type?

If no, then we can have one big metric map that includes both v7 and non-v7 metrics and we let the contents of the payload determine which metrics get submitted.